src/neptune/metadata_containers/project.py

```diff
@@ -124,16 +125,17 @@ def _prepare_nql_query(ids, states, owners, tags):
             )
 
         if states:
+            api_states = list(map(lambda s: RunState(s).to_api(), states))
             query_items.append(
                 NQLQueryAggregate(
                     items=[
                         NQLQueryAttribute(
                             name="sys/state",
                             type=NQLAttributeType.EXPERIMENT_STATE,
                             operator=NQLAttributeOperator.EQUALS,
-                            value=state,
+                            value=api_state,
```

Comment:
What about just `value=RunState(state).to_api()` instead of separate list?

Reply:
Good point.

I've implemented it higher and then moved the code here.